frameworks/digital-outcomes-and-specialists/messages/homepage-sidebar.yml

```diff
@@ -8,16 +8,19 @@ coming:
     You can apply to provide:
 
       - digital outcomes,
-        eg a discovery phase or an online billing application
+        eg a discovery phase for an online billing application
       - digital specialists,
         eg service managers or developers
       - user research studios
       - user research participants
 
-    [Create an account](https://www.digitalmarketplace.service.gov.uk/suppliers/create) to receive an email when Digital Outcomes and Specialists opens.
-
-
-    [Find out how to apply](https://www.digitalmarketplace.service.gov.uk/)
+    [Create an account](https://www.digitalmarketplace.service.gov.uk/suppliers/create) to receive an email when Digital Outcomes and Specialists opens. 
```

Comment:
Spoke to Sue. She recommended keeping it as two statements. Have made a small tweak to the second line here: 82c250f